lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -21,7 +21,8 @@ sub new {
 
 sub expect_is_shown {
     my ($self) = @_;
-    send_key_until_needlematch($self->{tag_agama_installer_highlighted}, 'down') unless check_screen($self->{tag_agama_installer_highlighted}, 5);
+
```

Comment:
no need to add blank line if the file doesn't follow that pattern.
8 would be better than 10, otherwise we could overpass the grub timeout of 10 seconds...

Reply:
remove the blank line and run with [8 secs](https://openqa.suse.de/tests/18753083)

Reply:
but 8 seconds will cause this [boot_from_hdd fails](https://openqa.suse.de/tests/18753093#step/agama_boot_from_hdd/38)